src/containers/SignIn/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import firebase from 'firebase/app';
+import 'firebase/auth';
+import { setError } from '../../actions';
+import Profile from '../Profile';
+
+export class SignIn extends Component {
+  constructor() {
+    super();
+    this.state = {
+      hasAuthorizedWithGitHub: false,
+      user: {}
+    };
+  }
+
+  handleSignIn = async () => {
```

Comment:
Great job! Glad we have the user images now so we can display those instead of just the filler image we have now